hic_qc.py

```diff
@@ -279,6 +284,9 @@ def extract_header_info(self, header, xx=50):
                 self.contigs_greater_10k (set(str)): The set of names of contigs with length > 10Kbp
                 self.contigs_greater_5k (set(str)): The set of names of contigs with length > 5Kbp
                 self.contigs_greater (dict(int-->set(str))): Dictionary with minimum lengths as keys and sets of contigs as values
+                self.command_line(str) : Full command-line argument used for alignment
+                self.bwa_command(str) : Subset of self.command_line containing only the BWA options used
+                self.samblaster(str) : Command used by samblaster
```

Comment:
Slight formatting difference here. Not blocking, but good to be consistent when possible.  No space is needed before the : in these lines.  Space before (str) is not consistent.  Either always do space or no space.

Again, not blocking, but wanted to point out.

Reply:
Fixing